docs/source/specifications/projjson.rst

```diff
@@ -71,19 +119,123 @@ The high level objects are:
   - ``Datum``
   - ``DatumEnsemble``
 
+.. _identifiers:
+
+Identifiers
++++++++++++
+
+All above mentioned objects can have an option ``id`` or ``ids`` member.
+
+The value of ``id`` is a JSON object with the following members:
+
+- ``authority``: (required) value of type string. e.g "EPSG", "OGC", "IGNF", etc.
+- ``code``: (required) value of type string or integer. e.g 4326 or "CRS84"
+- ``authority_citation``: (optional) value of type string that may be used to give
+  further details of the authority.
+- ``uri``: (optional) value of type string that may be used to give reference
+  to an online resource.
+
+An object can sometimes be identified in different ways, in which case the ``ids``
+member can be used to specify a JSON array of objects with the same type of ``id``.
+
+Identifiers are allowed in top-level objects and inner objects. The WKT2:2019
+specification recommends that if an object has an identifier, its inner objects
+should omit their identifiers, with the exceptions mentioned at
```

Comment:
and also above, right?

Reply:
> and also above, right?

what do you mean ? The doc doesn't say before that paragraph when an id can be used.

Reply:
Oops - sorry - I was confused.  Ignore this comment.